content/en/tracing/trace_collection/dd_libraries/java.md

```diff
@@ -137,6 +102,39 @@ For other environments, please refer to the [Integrations][5] documentation for
 {{% /tab %}}
 {{< /tabs >}}
 
+### Instrument Your Application
+
+Once the agent is installed, follow the instructions below to begin tracing your applications:
+
+1. Download `dd-java-agent.jar` that contains the latest Agent class files:
+
+   ```shell
+   wget -O dd-java-agent.jar https://dtdg.co/latest-java-tracer
+   ```
+   To access a specific tracer version, visit Datadog's [Maven repository][4].
+
+2. To run your app from an IDE, Maven or Gradle application script, or `java -jar` command, with the Continuous Profiler, deployment tracking, logs injection (if you are sending logs to Datadog), and trace volume control, add the `-javaagent` JVM argument and the following configuration options, as applicable:
```

Comment:
I don't see anything in the command below that is specific to `trace volume control`. That can probably be left out of this sentence to avoid confusion

Reply:
```suggestion
2. To run your app from an IDE, Maven or Gradle application script, or `java -jar` command, with the Continuous Profiler, deployment tracking, and logs injection (if you are sending logs to Datadog), add the `-javaagent` JVM argument and the following configuration options, as applicable:
```